app/src/main/kotlin/com/wire/android/ui/authentication/create/code/CreateAccountCodeViewModel.kt

```diff
@@ -209,7 +210,8 @@ class CreateAccountCodeViewModel @Inject constructor(
         clientScopeProviderFactory.create(userId).clientScope.getOrRegister(
             RegisterClientUseCase.RegisterClientParam(
                 password = password,
-                capabilities = null
+                capabilities = null,
+                modelPostfix = if(BuildConfig.PRIVATE_BUILD) " [${BuildConfig.FLAVOR}_${BuildConfig.BUILD_TYPE}]" else null
```

Comment:
Is PrivateBuild the correct parameter? Won't clients with custom builds also have that?

Reply:
Nevermind.

It seems we use `PRIVATE_BUILD` just for debug stuff. Only visible to us.

I guess we could name it better 🤔 
Anyway. LGTM!

Reply:
Just checked and as default we go with `false`, if not set (like the case of those custom builds)
https://github.com/wireapp/wire-android/blob/0f6bace635f79c3ee6ec416afe15759e97153b40/buildSrc/src/main/kotlin/scripts/variants.gradle.kts#L178-L183